### PR TITLE
[OCM-5426] Add the /notify_details endpoint to the SDK

### PR DESCRIPTION
**Pull request by Lir10**

### CODE REVIEW

**Comment by gshilin-sdb** [comment]
lgtm

**Comment by Lir10** [comment]
@tzvatot @zgalor  ptal

File: model/accounts_mgmt/v1/notify_details_request_type.model [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This struct is a request to get a templated email to a user related to this
+// subscription/cluster.
+struct NotificationDetailsRequest {
+	// Indicates which Cluster (internal id) the resource type belongs to
```
Comment (zgalor) [inline model/accounts_mgmt/v1/notify_details_request_type.model:20]: missing '.' at the end of each comment

Comment (Lir10) [reply model/accounts_mgmt/v1/notify_details_request_type.model]: added

File: model/accounts_mgmt/v1/notify_details_resource.model [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the notify endpoint.
+resource NotifyDetails {
+	// Post Notification details about user related to subscription/cluster via email and get the data associated with it.
+	method Post {
```
Comment (zgalor) [inline model/accounts_mgmt/v1/notify_details_resource.model:20]: method should be `Add`

Comment (Lir10) [reply model/accounts_mgmt/v1/notify_details_resource.model:20]: can't use `Add` as we need to use two separate `in` and `out` structs

Comment (zgalor) [reply model/accounts_mgmt/v1/notify_details_resource.model:20]: Ack

**Review by zgalor — APPROVED** [review]
lgtm

**Pull request merged**